lib/build/build.js

```diff
@@ -473,6 +472,16 @@ class Build {
     }
 
     this.info(`built ${sha} successfully`)
+
+    let { absoluteUrl } = destination
+    if (!absoluteUrl.endsWith('/')) {
+      absoluteUrl = `${absoluteUrl}/`
+    }
+
+    return {
+      outputURL: `${absoluteUrl}${evaluatedPathInDest}`,
+      localDirectory: isRemote ? null : destDir
+    }
```

Comment:
this is why we separated "step output" from the return value: the caller from this step will need this data to store it in the job status